x-pack/plugins/lens/public/visualizations/xy/color_assignment.ts

```diff
@@ -161,7 +161,7 @@ export function getAssignedColorConfig(
       : undefined;
   return {
     columnId: accessor as string,
-    triggerIcon: assignedColor ? 'color' : 'disabled',
+    triggerIconType: assignedColor ? 'color' : 'disabled',
     color: assignedColor ?? undefined,
```

Comment:
no need for the columnId type assertion

Reply:
Fixed [235e89e](https://github.com/elastic/kibana/pull/147686/commits/235e89ec86bda109bca48c952192c1cf93ab0b44)